src/KeyCode.js

```diff
@@ -1,128 +1,119 @@
-export default (() => {
+import * as KeyCode from './KeyCode';
```

Comment:
This is kind of hackish but work fine. Exporting itself for a default export in the end.

Reply:
Haven't seen this self-import thing yet. LOL :smile:. Let's wait to see if all tests pass.

Reply:
All tests pass. 💚